src/gstreamer.rs

```diff
@@ -0,0 +1,416 @@
+use App;
+use RecordFormat;
+use SnapshotFormat;
+use SnapshotSettings;
```

Comment:
Maybe we can put all that settings related stuff in a single module? The enums, structs and trait impls

Reply:
I thought about it but didn't know if we wanted that.

Reply:
I think it would be an improvement

Reply:
Could also be moved to utils.rs and utils.rs though, where all the setting stuff is now anyway